src/main/java/org/apache/sling/event/impl/jobs/tasks/CleanUpTask.java

```diff
@@ -181,6 +181,11 @@ public void log(String message, Object... args) {
 
                         }
 
+                        @Override
+                        public void setProperty(String name, Object value) {
+
```

Comment:
please at least a comment here, not just a empty method body.

Reply:
All the methods of that JobExecutionContext had no comments.  So I addedd a comment at the top of the class explaining why.